openedx/core/djangoapps/discussions/models.py

```diff
@@ -89,6 +90,7 @@ class Features(Enum):
         )._asdict(),
     },
     'piazza': {
+        'additional_messages': [],
```

Comment:
TBH I don't see what use case is best served by a named tuple here; I'm looking at this as a group of pretty generic messages but you probably have something in mind I'm not aware of.

Reply:
what is the boundary for a message? A single <p> block of HTML? I expect only 1 message generally but was curious what the bounds of messages are today. 

Relatedly, can these messages be configured to include links or is this version just unformatted text? (ok if just unformatted text but I could imagine wanting to use links in the messages if ever a message requires external documentation or content)

Reply:
@xitij2000 I finally understood that you meant by creating a named tuple for different message types; that's a good idea, I'm going to create it with a `privacy` field.
I'm keeping each of these as a list though, simply because it's not much of an inconvenience to client code while leaving room for extension.
If you feel otherwise let me know and I'll change it back!

@marcotuts I've gone with unformatted text simply because any of the alternatives is substantially more complex and potentially insecure (e.g. returning markup directly or returning a data structure to be used for layout).
Not sure if there's anything under discussion yet but hopefully this could be the trigger for it :)

Reply:
@kewne It's OK to be a named tuple, but it should be retruned as an object/dict instead of a list.